pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
+ name = "mod_ca";
```

Comment:
For packages you should use `pname` in combination with `version` so that `name` is properly generated (as `name = pname + '-' + name`). So in this case you want
```
pname = "mod_ca";
version = "0.2.1";
```
and then `name` can be omitted.